node-notes-cli/app.js

```diff
@@ -0,0 +1,87 @@
+import { readFile, writeFile } from 'node:fs/promises';
+
+const argVar = process.argv[2];
+const content = process.argv[3];
+const content2 = process.argv[4];
+
+if (argVar === 'read' && content === undefined) {
+  try {
+    const response = await readFile('./data.json', 'utf8');
+    const data = JSON.parse(response);
+    if (data && data.notes) {
+      for (const key in data.notes) {
+        if (Object.getOwnPropertyDescriptor(data.notes, key)) {
```

Comment:
This check isn't necessary in modern JS since for-in only goes through "own" properties.